### PR TITLE
Design General Layout

### PR DESCRIPTION
**Pull request by Moonforeva**

### CODE REVIEW

**Comment by Moonforeva** [comment]
Ready

File: app/assets/stylesheets/colour.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+$blue: #0006FF;
+$red: #F90E05;
+$lightgrey: #eaeaea;
+$darkyellow: #777;
+$darkgrey: #555;
+$blackyellow: #222;
+
```
Comment (saobv-2223) [inline app/assets/stylesheets/colour.scss]:
```suggestion
$blackyellow: #222;

```

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    The <%= link_to "COURSE REGISTRATION SYSTEM", root_path %>
+    designed by Malay-Group-4 
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About", about_path %></li>
+      <li><%= link_to "Contact", contact_path %></li>
+    </ul>
+  </nav>
+</footer>
```
Comment (saobv-2223) [inline app/views/layouts/_footer.html.erb]: add endline

File: app/views/static_pages/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<html>
+  <head>
+    <title><%= yield(:title) %> | Course Registration System </title>
+  </head>
+  <body>
+    <h1>ABOUT </h1>
+    <h2>Company History</h2>
+    <p>This company is established by Mangkuk Hayun.Sdn.Bhd</p>
+  </body>
+</html>
```
Comment (saobv-2223) [inline app/views/static_pages/about.html.erb]: add endline

File: app/views/static_pages/contact.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Contact" %>
+<html>
+  <head>
+    <title><%= yield(:title) %> | Course Registration System </title>
+  </head>
+  <body>
+    <h1>If there are any problem, please contact </h1>
+    <p>Salehuddin Bin Abd Latif <%= link_to "Facebook", Settings.contacts.salehuddin %></p>
+  </body>
+</html>
```
Comment (saobv-2223) [inline app/views/static_pages/contact.html.erb]: add endline

File: config/settings.yml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+contacts:
+  salehuddin: https://www.facebook.com/dongdang.ding/
```
Comment (saobv-2223) [inline config/settings.yml]: add endline

**Comment by Moonforeva** [comment]
Ready

File: app/assets/stylesheets/colour.scss [hunk 6]
```diff
@@ -0,0 +1,6 @@
+$blue: #0006FF;
+$red: #F90E05;
+$lightgrey: #eaeaea;
+$darkyellow: #777;
+$darkgrey: #555;
+$blackyellow: #222;
```
Comment (saobv-2223) [inline app/assets/stylesheets/colour.scss]: add endline

File: config/settings.yml [hunk 7]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (saobv-2223) [inline config/settings.yml]:
```suggestion
  salehuddin: https://www.facebook.com/dongdang.ding/

```

**Comment by Moonforeva** [comment]
Ready

**Pull request merged**